runtime/js/jslib.js

```diff
@@ -146,8 +139,23 @@ function caml_jsoo_flags_use_js_string(unit) {
 }
 
 //Provides: caml_jsoo_flags_effects
+//If: !effects
+function caml_jsoo_flags_effects(unit) {
+  return "none";
+}
+
+//Provides: caml_jsoo_flags_effects
+//If: effects
+//If: !doubletranslate
+function caml_jsoo_flags_effects(unit) {
+  return "cps";
+}
+
+//Provides: caml_jsoo_flags_effects
+//If: effects
+//If: doubletranslate
 function caml_jsoo_flags_effects(unit) {
-  return FLAG("effects");
+  return "double-translation";
```

Comment:
Maybe we could update `macro.ml` to handle this case. We could have a `CONFIG` macro.